lib/editor/components/map/PatternStopsLayer.js

```diff
@@ -80,9 +81,8 @@ export default class PatternStopsLayer extends Component<Props> {
     return (
       <div id='PatternStops'>
         {patternHalts.map((patternStop, index) => {
-          if (patternStop.hasOwnProperty('stopId')) {
-            // $FlowFixMe Flow doesn't understand our type check
-            const { cpIndex, stopId } = patternStop
+          if (patternHaltIsStop(patternStop)) {
+            const { cpIndex, stopId } = patternHaltIsStop(patternStop)
```

Comment:
To avoid writing `FlowFixMe`. Otherwise flow doesn't understand that the boolean means we know the variable is a `patternStop` and we have to annotate almost every line. By returning a `patternStop` from the method we solve this problem, although we have to call the method twice. This felt cleaner to me than extracting it to a new variable

Reply:
Have extracted anyway to match the change in the comment